tools/ibc-test/README.md

```diff
@@ -36,13 +36,9 @@ cd $AXON_SRC_PATH
 git checkout forcerelay-test
 cargo install --path .
 
-# setup hermes envs
-## CKB
-export CHAIN_COMMAND_PATHS=ckb
-export ACCOUNT_PREFIXES=ckb
-## Axon
-export CHAIN_COMMAND_PATHS=axon
-export ACCOUNT_PREFIXES=axon
+# Chains to test (chain_a,chain_b). You can also use `axon,ckb`, `ckb,ckb` or `axon,axon`
```

Comment:
If ckb is involved, why not add the commands to install ckb.
Only axon is mentioned.

Reply:
Installation command will be different depending on OS and CPU. I'll add a brief mention of the ckb requirement.